base/bitset.jl

```diff
@@ -215,24 +230,24 @@ function _matched_map!(f, a1::Bits, b1::Int, a2::Bits, b2::Int,
             # update ediff in case l1 was updated
             ediff = e2 - l1 - b1
             # copy actual chunks from a2
-            unsafe_copyto!(a1, l1+1, a2, l2+1-ediff, ediff)
+            unsafe_copyto!(a1, l1 + 1, a2, l2 + 1 - ediff, ediff)
             l1 = length(a1)
         end
     elseif ediff < 0
         if right_false_is_false
             # We don't need to worry about the trailing bits — they're all false
             _deleteend!(a1, min(l1, -ediff))
-            # no need to update l1, as if bdiff > 0 (case below), then bdiff will
-            # be smaller anyway than an updated l1
+        # no need to update l1, as if bdiff > 0 (case below), then bdiff will
+        # be smaller anyway than an updated l1
         else # @assert f(x, false) == x
-            # We don't need to worry about the trailing bits — they already have the
-            # correct value
+        # We don't need to worry about the trailing bits — they already have the
+        # correct value
         end
     end
 
     if bdiff < 0
         if left_false_is_false
-            # We don't need to worry about the leading bits — they're all false
+        # We don't need to worry about the leading bits — they're all false
```

Comment:
huh?

Reply:
Yeah it doesn't mess with your comments anymore, was trying to be too fancy before